jobs/config.json

```diff
@@ -8165,6 +8165,28 @@
       "sig-cluster-lifecycle"
     ]
   },
+  "ci-kubernetes-e2e-kubeadm-gce-dns-coredns": {
+    "args": [
+      "--cluster=",
+      "--deployment=kubernetes-anywhere",
+      "--env-file=jobs/platform/gce.env",
+      "--extract=ci/latest",
+      "--gcp-project=ib-cloudteam-dev-148222",
```

Comment:
Just want to make sure that it's intentional that you always want to use this project, and not one of the projects in the Boskos pool. Whoever owns the project will need to take care of billing and making sure that the prow service account has permissions to be able to manage clusters there.

Reply:
right that is an error. this projects was for my tests. locally. removing